luminaire/model/window_density.py

```diff
@@ -437,18 +393,14 @@ def train(self, data, **kwargs):
         >>> success, model = wdm_obj.train(data)
 
         >>> success, model
-        (True, <luminaire.model.window_density.WindowDensityModel object at 0x7fd7c5a34e80>)
+        (True, "2018-10-10 23:00:00", <luminaire.model.window_density.WindowDensityModel object at 0x7fd7c5a34e80>)
         """
         import numpy as np
         import pandas as pd
 
-        freq = self._params['freq']
-        min_num_train_windows = self.min_num_train_windows
-        max_num_train_windows = self.max_num_train_windows
-        ignore_window = self._params['ignore_window']
+        freq = pd.Timedelta(self._params['freq']) if self._params['freq'] not in ['S', 'T', '15T', 'H', 'D'] \
```

Comment:
We should create a list variable for such time frequencies, to make it easier to update code in future.

Reply:
The frequency can be arbitrary for this model. I just listed some typical ones to apply the corresponding settings.